### PR TITLE
'add tests for `commands/initialize/createDeployKey`'

### PR DESCRIPTION
**Pull request by jacobroufa**

### CODE REVIEW

File: tests/unit/src/commands/initialize/createDeployKey.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+let createKeyStub: SinonStub;
+let encryptDataStub: SinonStub;
+let decryptDataStub: SinonStub;
+let equalStub: SinonStub;
+
+registerSuite({
+	name: 'commands/initialize/createDeployKey',
+
+	before() {
+		encryptedStub = {
```
Comment (devpaul) [inline tests/unit/src/commands/initialize/createDeployKey.ts]: move these definitions w/ their declarations and make them `const`

File: tests/unit/src/commands/initialize/createDeployKey.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+				assert.isTrue(encryptedKeyFileStub.calledOnce);
+			},
+
+			async 'Deploy key already exists'() {
+				existsSyncStub.returns(true);
+
+				try {
+					await createDeployKey('deploykey.file', 'deploykey.enc');
+				} catch (err) {
+					assert.strictEqual(err.message, 'Deploy key already exists');
```
Comment (devpaul) [inline tests/unit/src/commands/initialize/createDeployKey.ts]: If createDeployKey doesn't throw, then this test will pass. 

these assertions need to be moved below the catch statement or this statement needs to be returned to intern as a promise like:

```
const promise = await createDeployKey('deploykey.file', 'deploykey.enc');

return promise.then(assert.fail, (err) => { /* assertions */ });
```

File: tests/unit/src/commands/initialize/createDeployKey.ts (hunk 1 above)
Comment (jacobroufa) [inline tests/unit/src/commands/initialize/createDeployKey.ts]: addressed in a1b4343

File: tests/unit/src/commands/initialize/createDeployKey.ts (hunk 2 above)
Comment (jacobroufa) [inline tests/unit/src/commands/initialize/createDeployKey.ts]: addressed in a1b4343

**Pull request merged**